app/src/main/java/org/p2p/wallet/newsend/NewSendPresenter.kt

```diff
@@ -1,5 +1,45 @@
 package org.p2p.wallet.newsend
 
+import kotlinx.coroutines.launch
+import org.p2p.wallet.common.analytics.interactor.ScreensAnalyticsInteractor
 import org.p2p.wallet.common.mvp.BasePresenter
+import org.p2p.wallet.home.analytics.BrowseAnalytics
+import org.p2p.wallet.home.model.Token
+import org.p2p.wallet.user.interactor.UserInteractor
+import kotlin.properties.Delegates
 
-class NewSendPresenter() : BasePresenter<NewSendContract.View>(), NewSendContract.Presenter
+class NewSendPresenter(
+    private val userInteractor: UserInteractor,
+    private val browseAnalytics: BrowseAnalytics,
+    private val analyticsInteractor: ScreensAnalyticsInteractor,
+) : BasePresenter<NewSendContract.View>(), NewSendContract.Presenter {
+
+    private var token: Token.Active? by Delegates.observable(null) { _, _, newValue ->
```

Comment:
tokenToSend

Reply:
no it is selected token in common

Reply:
but we don't send it? what this token for?

Reply:
override fun setTokenToSend(newToken: Token.Active) {
        token = newToken
    }